### PR TITLE
Return code interpreter example to use exec

### PR DESCRIPTION
**Pull request by pawalt**
Previously we had a bug in exec that made it drop some messages. This bug is fixed so we can return to our preferred exec pattern.

### Type of Change

- [ ] New example
- [x] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

## Checklist

- [x] Example is testable in synthetic monitoring system, or `lambda-test: false` is added to example frontmatter (`---`)
  - [x] Example is tested by executing with `modal run` or an alternative `cmd` is provided in the example frontmatter (e.g. `cmd: ["modal", "deploy"]`)
  - [x] Example is tested by running with no arguments or the `args` are provided in the example frontmatter (e.g. `args: ["--prompt", "Formula for room temperature superconductor:"]`
- [x] Example is documented with comments throughout, in a [_Literate Programming_](https://en.wikipedia.org/wiki/Literate_programming) style.
- [x] Example does _not_ require third-party dependencies to be installed locally
- [x] Example pins its dependencies
  - [x] Example pins container images to a stable tag, not a dynamic tag like `latest`
  - [x] Example specifies a `python_version` for the base image, if it is used
  - [x] Example pins all dependencies to at least minor version, `~=x.y.z` or `==x.y`
  - [x] Example dependencies with `version < 1` are pinned to patch version, `==0.y.z`

### CODE REVIEW

**Comment by pawalt** [comment]
@prbot automerge

**Comment by pawalt** [comment]
whoops i guess automerge doesn't work on this repo

**Comment by charlesfrye** [comment]
🚀 The docs preview is ready! Check it out here:     https://modal-labs-examples--frontend-preview-b91f452.modal.run

File: 13_sandboxes/simple_code_interpreter.py [hunk 1]
```diff
@@ -67,11 +67,11 @@ def driver_program():
 # that program and execute code in it.
```
Comment (charlesfrye) [inline 13_sandboxes/simple_code_interpreter.py:67]: @pawalt could you also update the text here

**Pull request merged**